index.html

```diff
@@ -39,7 +39,7 @@
         <h2>
 <a id="welcome-to-yoga-with-allison" class="anchor" href="#welcome-to-yoga-with-allison" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Welcome to Yoga with Allison</h2>
 
-<p>I studied at the Himalayan Institutue of Buffalo. I am a 200 hour Registered Yoga Teacher.</p>
+<p>I am a certified 200 hour yoga teacher, with training through at the Himalayan Institutue of Buffalo, and accredited by Yoga Alliance as a 200 hr RYT.</p>
```

Comment:
It's also kind of redundant in my opinion, but you did the same thing on your business cards.